<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>160</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="actionButtonsLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="btnAddItem">
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnUpdateItem">
       <property name="text">
        <string>update</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnDeleteItem">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnClearFields">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>160</width>
      <height>204</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_modifikasiBarang">
     <item row="0" column="0">
      <widget class="QLabel" name="id_label">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditId">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nama_label">
       <property name="text">
        <string>Nama</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="price_label">
       <property name="text">
        <string>Harga</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditPrice">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="quantity_label">
       <property name="text">
        <string>Jumlah</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditQuantity">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="appTitle_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Simple Inventory Management App</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="daftarBarang_label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>daftar barang</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="searchId_label">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSearchId">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSearchItem">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="pencarianBarang_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>pencarian barang</string>
    </property>
   </widget>
   <widget class="QLabel" name="modifikasiBarang_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>modifikasi barang</string>
    </property>
   </widget>
   <zorder>tableWidget</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>formLayoutWidget_2</zorder>
   <zorder>appTitle_label</zorder>
   <zorder>daftarBarang_label</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>pencarianBarang_label</zorder>
   <zorder>modifikasiBarang_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
